AWSIoT/AWSIoTDataManager.m

```diff
@@ -912,22 +912,39 @@ - (void)createSubscriptionsForShadow:(AWSIoTDataShadow *)shadow
  */
 - (BOOL)handleSubscriptionsForShadow:(NSString *)name
                             callback:(AWSIoTMQTTExtendedNewMessageBlock)callback {
+    return [self handleSubscriptionsForShadow:name callback:callback completionHandler:nil];
+}
+
+- (BOOL)handleSubscriptionsForShadow:(NSString *)name
```

Comment:
nit: can we have a doc comment like the above function?

Reply:
will add!